### PR TITLE
Add FundingTarget Page

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

_Provide a description of what this PR does. Link to any relevant GitHub issues, Notion tasks or Discord discussions._

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1970** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1970" target="_blank">Graphite</a> 
    * **PR #1971** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1971" target="_blank">Graphite</a> 
      * **PR #1972** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1972" target="_blank">Graphite</a> 
        * **PR #1973** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1973" target="_blank">Graphite</a> 
          * **PR #1974** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1974" target="_blank">Graphite</a> 
            * **PR #1979** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1979" target="_blank">Graphite</a> 
              * **PR #1985** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1985" target="_blank">Graphite</a> 
                * **PR #2004** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2004" target="_blank">Graphite</a> 
                  * **PR #2005** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2005" target="_blank">Graphite</a> 
                    * **PR #2006** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2006" target="_blank">Graphite</a> 
                      * **PR #2007** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2007" target="_blank">Graphite</a> 
                        * **PR #2008** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2008" target="_blank">Graphite</a> 
                          * **PR #2012** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2012" target="_blank">Graphite</a> 
                            * **PR #2013** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2013" target="_blank">Graphite</a> 
                              * **PR #2014** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2014" target="_blank">Graphite</a> 
                                * **PR #2015** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2015" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2015?utm_source=stack-comment).

File: src/components/Create/Create.tsx [hunk 1]
```diff
@@ -46,6 +47,13 @@ export function Create() {
             >
               <FundingCyclesPage />
             </Wizard.Page>
+            <Wizard.Page
+              name="fundingTarget"
+              title={t`Funding Target`}
+              description={t`Select an option that best suits your project’s funding requirements.`}
```
Comment (tomquirk) [inline src/components/Create/Create.tsx:53]:
```suggestion
              description={t`Select the option that best suits your project’s funding requirements.`}
```

File: src/components/Create/components/pages/FundingTarget/FundingTargetPage.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        <Form.Item noStyle name="targetSelection">
+          <Selection>
+            <Selection.Card
+              name="specific"
+              title={t`Specific Funding Target`}
+              icon={<PushpinOutlined />}
+              description={
+                <Trans>
+                  Set a specific amount that you will be able to distribute to
+                  nominated wallets each funding cycle.
```
Comment (tomquirk) [inline src/components/Create/components/pages/FundingTarget/FundingTargetPage.tsx:45]:
```suggestion
                  Set a specific amount to distribute to
                  nominated wallets each funding cycle.
```

File: src/components/Create/components/pages/FundingTarget/FundingTargetPage.tsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+                </Trans>
+              }
+            >
+              <Form.Item
+                name="amount"
+                label={t`Set your funding target`}
+                extra={
+                  <Trans>
+                    Note: your project’s settings will not be able to be edited
```
Comment (tomquirk) [inline src/components/Create/components/pages/FundingTarget/FundingTargetPage.tsx:54]: **sugetion** maybe info icon here too

File: src/components/Create/components/pages/FundingTarget/FundingTargetPage.tsx [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+            </Selection.Card>
+            <Selection.Card
+              name="infinite"
+              title={t`Infinite Funding Target`}
+              icon={<RetweetOutlined />}
+              description={
+                <Trans>
+                  All funds raised will be available to distribute to payout
+                  addresses. Your project will have no overflow, so token
+                  holders will not be able to redeem.
```
Comment (tomquirk) [inline src/components/Create/components/pages/FundingTarget/FundingTargetPage.tsx:70]:
```suggestion
                  Your project will withhold all funds raised. Funds can be distributed to payout
                  addresses you set, and the project owner will receive any unallocated funds. Your project will have no overflow, so token
                  holders can't redeem their tokens for ETH.
```

File: src/components/Create/components/pages/FundingTarget/FundingTargetPage.tsx [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+              }
+            />
+            <Selection.Card
+              name="none"
+              title={t`No Funding Target`}
+              icon={<CloseCircleOutlined />}
+              description={
+                <Trans>
+                  All funds raised will be considered ‘overflow’ and will be
+                  redeemable by token holders at any time.
```
Comment (tomquirk) [inline src/components/Create/components/pages/FundingTarget/FundingTargetPage.tsx:81]:
```suggestion
                  Your project won't withhold any funds. All funds raised are Overflow. Token holders can redeem their tokens for a portion of your entire treasury.
```

File: src/components/Create/components/pages/FundingTarget/hooks/FundingTargetForm.ts [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+      amount: { amount: distributionLimit.toString(), currency },
+    }
+  }, [fundAccessConstraints])
+
+  const dispatch = useAppDispatch()
+  const targetSelection = useWatch('targetSelection', form)
+  const amount = useWatch('amount', form)
+
+  useEffect(() => {
+    if (targetSelection === 'infinite') {
```
Comment (tomquirk) [inline src/components/Create/components/pages/FundingTarget/hooks/FundingTargetForm.ts:104]: bro CLEAN UP ON AISLE 6!!! clean af, ty ty

File: src/components/Create/components/pages/FundingTarget/hooks/FundingTargetForm.ts [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+    const distributionLimit = BigNumber.from(
+      fundAccessConstraints[0].distributionLimit,
+    )
+    if (distributionLimit.eq(MAX_DISTRIBUTION_LIMIT)) {
+      targetSelection = 'infinite'
+    } else if (distributionLimit.eq(0)) {
+      targetSelection = 'none'
+    } else {
+      targetSelection = 'specific'
+    }
```
Comment (tomquirk) [inline src/components/Create/components/pages/FundingTarget/hooks/FundingTargetForm.ts:85]: nailed it

**Review by tomquirk — APPROVED** [review]
only copy suggestions, everything else perfect

**Pull request merged**